client/src/components/auth/VerifiedEmailComponent.vue

```diff
@@ -0,0 +1,24 @@
+<template>
+    <div></div>
```

Comment:
add localization with text: "Your account is being activated..." and if success "Your account has been activated. Please, log in" and "log in" make as link to log in page